openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/GlossaryPagination.spec.ts

```diff
@@ -191,3 +198,129 @@ test.describe('Glossary tests', () => {
     await page.waitForResponse('api/v1/glossaryTerms?*');
   });
 });
+
+test.describe('Pagination tests for all pages', () => {
+  test('should test pagination on Users page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto('/settings/members/users');
+    await testPaginationNavigation(page, 'table');
+  });
+
+  test('should test pagination on Roles page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto('/settings/access/roles');
+    await testPaginationNavigation(page, 'table');
+  });
+
+  test('should test pagination on Policies page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto('/settings/access/policies');
+    await testPaginationNavigation(page, 'table');
+  });
+
+  test('should test pagination on Database Services page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto(
+      '/databaseSchema/sample_data.ecommerce_db.shopify?showDeletedTables=false'
+    );
+    await testPaginationNavigation(page, 'table');
+  });
+
+  test('should test pagination on Bots page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto('/settings/bots');
+    await testPaginationNavigation(page, 'table');
+  });
+
+  test('should test pagination on Service version page', async ({ page }) => {
+    test.slow(true);
+
+    await page.goto(`/service/dashboardServices/sample_superset/versions/0.1`);
+    await testPaginationNavigation(page, 'table');
+  });
+});
+
+test.describe('Pagination tests for Classification Tags page', () => {
+  const classification = new ClassificationClass();
+  const tags: TagClass[] = [];
+
+  test.beforeAll(async ({ browser }) => {
+    const { apiContext, afterAction } = await createNewPage(browser);
+    await classification.create(apiContext);
+
+    for (let i = 1; i <= 20; i++) {
+      const tag = new TagClass({
+        classification: classification.responseData.name,
+        name: `pw-tag-pagination-${uuid()}-${i}`,
+        displayName: `PW Tag Pagination ${i}`,
+        description: `Tag ${i} for pagination testing`,
+      });
+      await tag.create(apiContext);
+      tags.push(tag);
+    }
+
+    await afterAction();
+  });
+
+  test.afterAll(async ({ browser }) => {
+    test.setTimeout(8 * 60 * 1000);
+
+    const { apiContext, afterAction } = await createNewPage(browser);
+
+    for (const tag of tags.reverse()) {
+      await tag.delete(apiContext);
+    }
```

Comment:
deleting clarification should delete all tags, and if its not happening then its bug